i18n/en.json

```
@@ -2943,6 +2943,8 @@
   "claim.oauth_to_email.pwdNotMatch": "Passwords do not match.",
   "claim.oauth_to_email.switchTo": "Switch {type} to Email and Password",
   "claim.oauth_to_email.title": "Switch {type} Account to Email",
+  "cloud_archived.error.access": "Permalink belongs to a message that has been archived because of cloud plan limits. Upgrade cloud plan to access message again.",
```

nit:
```suggestion
  "cloud_archived.error.access": "Permalink belongs to a message that has been archived because of {plan} limits. Upgrade cloud plan to access message again.",
```

I think this would read better